index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="styles.css" rel="stylesheet">
+  <title>Final Project</title>
+</head>
+<body>
+  <header>
+    <nav class="nav__bar" id="navbar">
+      <ul>
+        <li><a class="nav__links" href="#welcome-section">About</a></li>
+        <li><a class="nav__links" href="#projects">Work</a></li>
+        <li><a class="nav__links" href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+    <section class="section__first" id="welcome-section">
+      <div class="container">
+      <h1 class="section__h1">Hey I am Nooria</h1>
+      </div>
+      <p class="section__p">a web developper</p>
+    </section>
+  </header>
+<main>
+  <section class="section__projects" id="projects">
+    <h2 class="text-h2">These are some of my projects</h2>
+    <div class="projects__div">
+      <a href="https://github.com/Nooria150/tribute-page" target="_blank" class="project-tile" rel="noopener">
+        <img class="projects__images" src="images/tribute.PNG">
+        <p class="projects__p">Tribute Page</p>
+      </a>
+      <a href="https://github.com/Nooria150/technical-documentation-page" target="_blank" class="project-tile" rel="noopener">
+        <img class="projects__images" src="images/WhatsApp Image 2024-05-27 at 12.25.37_3977bfde.jpg">
+        <p class="projects__p">technical documentaion</p>
+      </a>
+      <a href="https://github.com/Nooria150/product-landing-page" target="_blank" class="project-tile" rel="noopener">
+        <img class="projects__images" src="images/_C__Users_DELL_Desktop_product-landing-page_index.html (3).png">
+        <p class="projects__p">product landing</p>
+      </a>
+    </div>
+    <div class="div__button">
+      <button class="projects__button">Show all &nbsp;&nbsp;></button>
+    </div>
+  </section>
+</main>
+<section class="section__contact" id="contact">
+  <div class="contact__header">
+    <h2 class="contact__h2">Let's work together...</h2>
+    <p class="contact__p">How do you take your coffee?</p>
+  </div>
+  <div class="contact__links">
+    <a href="https://github.com/Nooria150" id="profile-link" target="_blank" rel="noopener">Github</a>
+    <a href="nooriamangal55@gmail.com" id="profile-link" target="_blank">Send an Email</a>
+    <a href="555 555 555" id="profile-link" target="_blank">Call me</a>
+  </div>
+</section>
+<footer class="footer">
+  <p class="wrap inline">This portfolio is for practic.</p>
+</footer>
```

Comment:
- These codes all are inside your body si they all should be 2 spaces ahead of your body tag